README.md

```diff
@@ -78,6 +78,17 @@ $ env CI=true npm test
 ```
 This behaves similarly to running `jest --ci`.
 
+### Jest Configuration Mode (Recommended for Jest Users)
+
+If you are using Jest, but prefer Chai assertions, you don’t have anything to configure except loading the plugin itself: the `matchSnapshot` Chai assertion will automatically delegate to Jest’s built-in snapshot capability, so all usual options, settings, CLI flags, method arguments, etc. will work out of the box.
+
+```js
+import chai from "chai";
+import chaiJestSnapshot from "chai-jest-snapshot";
+
+chai.use(chaiJestSnapshot);
+```
+
 ### Framework-agnostic Configuration Mode (Recommended for Non-Mocha Users)
 If you are not using mocha as your test runner, it is recommended to use chai-jest-snapshot in "framework-agnostic configuration mode".
```

Comment:
Let's update this to say "If you are not using mocha or jest as your test runner"